models/user/user.go

```diff
@@ -1232,3 +1232,21 @@ func GetOrderByName() string {
 	}
 	return "name"
 }
+
+// FeatureDisabledWithLoginType checks if a user feature is disabled, taking into account the login type of the
+// user if applicable
+func FeatureDisabledWithLoginType(user *User, feature string) bool {
```

Comment:
Rename to `IsFeatureDisabledWithLoginType`, boolean-returning functions should always be prefixed with `Is` in my book.